src/electron/main/main.ts

```diff
@@ -56,6 +58,13 @@ const createWindow = () => {
         nativeHighContrastModeListener.startListening();
     });
 
+    mainWindowRendererMessageHandlers = new MainWindowRendererMessageHandlers(
+        app,
+        mainWindow,
+        ipcMain,
+    );
+    mainWindowRendererMessageHandlers.startListening();
```

Comment:
Should this be waiting to set up listeners until the IPC_MAIN_WINDOW_INITIALIZED_CHANNEL_NAME event, like the native HC listener works?

Reply:
Curiously, that didn't work. I think it's because the getVersion() call is synchronous and causing a deadlock. The main window object has been created at this point, so it's safe to install the listeners before the window is fully initialized.